pkgs/top-level/php-packages.nix

```diff
@@ -1,10 +1,12 @@
 { pkgs, php }:
 
-let self = with self; {
-  buildPecl = import ../build-support/build-pecl.nix {
-    inherit php;
-    inherit (pkgs) stdenv autoreconfHook fetchurl;
-  };
+let
+  self = with self; {
+    buildPecl = import ../build-support/build-pecl.nix {
+      inherit php;
+      inherit (pkgs) stdenv autoreconfHook fetchurl;
+    };
+  php7 = pkgs.lib.versionAtLeast php.version "7.0";
```

Comment:
Does this need to be in phpPackages? Also, it should be named something like `isPhp7` or similar, `php7` makes me think it's a package.